### PR TITLE
Profiles: fix update actions in gestures

### PR DESCRIPTION
**Pull request by hius07**
Fixes https://github.com/koreader/koreader/issues/11161#issuecomment-1827692855.

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/11175)

### CODE REVIEW

**Comment by hius07** [comment]
Currently there is no crash if there are no custom multiswipes.

**Comment by Frenzie** [comment]
I mean if they will apply and aren't filtered out or something.

**Comment by hius07** [comment]
Sorry, don't understand the question.

File: plugins/gestures.koplugin/main.lua [hunk 1]
```diff
@@ -1216,7 +1216,8 @@ function Gestures:onFlushSettings()
 end
 
 function Gestures:updateProfiles(action_old_name, action_new_name)
-    for section, gestures in pairs(self.settings_data.data) do -- custom_multiswipes, gesture_fm, gesture_reader
```
Comment (Frenzie) [inline plugins/gestures.koplugin/main.lua:1219]: The old comment seems to imply the way it's written there is necessary for custom multiswipes to function in this context, which is a bit of a separate question from if it crashes or not. ^^

Comment (hius07) [reply plugins/gestures.koplugin/main.lua:1219]: Got it, "they" means "custom multiswipes".
Yes, they will still work properly.
By (unexpected) design, the custom_multiswipes section in gestures.lua has different format and doesn't contain actions. All actions for custom multiswipes are stored in fm and reader sections, and will be updated properly when some actions are added to or removed from the dispatcher list.

**Pull request merged**